resources/webroot/index.js.mustache

```diff
@@ -78,7 +78,7 @@ let availableItems = []
 let availableTileservers = {}
 let selectedTileserver = 'Default'
 let availableIconStyles = {}
-let selectedIconStyle = 'Default'
+let selectedIconStyle
```

Comment:
Shouldn't this be
```let selectedIconStyle = '{{default_icon_style}}'```
?

Reply:
It's set during `function loadStorage()` - availableIconStyles does contain the default one too, why 2 params? :)

Reply:
fixed that in latest commit